docs/tutorials/did-web-discovery/README.md

```diff
@@ -1,9 +1,157 @@
-In this tutorial we will test:
+# DID Web Discovery Tutorial
 
-1. verifying did-web resolution
+If you have not done so already, please review and complete the previous tutorials which form the foundation for the material presented in this tutorial.
+
+1. [Access Token Tutorial](../authentication/README.md) - Obtain an OAuth2 client credentials grant token
+
+In this tutorial, we will be using DID Web Discovery to find the base URL for a VC-API that supports presentation exchange as well as a DID to use for issuing credentials.
+
+## Setup and Configuration
+
+This tutorial will be picking up where the Access Token Tutorial left off; the quickest way to get set up to work on this tutorial is to:
+
+1. Duplicate the Access Token Tutorial and rename it to DID Web Discovery Tutorial.
+1. Duplicate the Access Token Environment and rename it to DID Web Discovery Environment.
+
+Be sure to activate the new DID Web Discovery Environment by selecting it in the drop-down menu at the top-right of the Postman interface.
+
+<img src="./resources/select-environment.png"/>
+
+## Access Token Request
+
+The first thing that we need to do is preserve the output of the Get Access Token request so that we can use the created access token for authenticated requests to the VC API endpoint.
```

Comment:
```suggestion
First, preserve the output of the `Get Access Token` request, so that we can use the created access token for authenticated requests to the VC API endpoint.
```